docs/index.rst

```diff
@@ -356,6 +356,28 @@ An example that combines a dbname with a query string::
 
    memory://storagename?connection_cache_size=100&database_name=fleeb
 
+
+``demo:`` URI scheme
+~~~~~~~~~~~~~~~~~~~~
+
+The ``demo:`` URI scheme can be passed as ``zodbconn.uri`` to create a
+DemoStorage database factory. DemoStorage provides an overlay combining base
+and δ storages. The URI scheme contains two parts - base and δ::
```

Comment:
What is `δ`? Would you please spell this out and define its first occurrence? It's fine to refer to it as the symbol only after its definition.

Reply:
ok, I've rewored this text to be

```
DemoStorage provides an overlay combining base
and changes storages. The URI scheme contains two parts - base and δ for changes::
```

Reply:
Sorry, I wasn't clear. I do not know what is this symbol: `δ`. What does it mean?

For example, `®` means "registered trademark".

Reply:
I see now. δ is just a greek letter "delta" usually referring to some changes in math, physics etc. Is it ok this way? Or should we change and just spell everything as "changes" literally?

Reply:
Ah, OK, I used to write it in upper case `Δ`, or sloppier as a lower case `d` without the flag. I think to make it clear, we can define the symbol once upon its first use, and use only the symbol thereafter. I'll make a separate suggestion.

Reply:
( thanks, commented [there](https://github.com/Pylons/zodburi/pull/29#discussion_r594148029) )